.opex/external/env/prod/backend.tfvars

```diff
@@ -0,0 +1,4 @@
+resource_group_name  = "terraform-state-rg"
+storage_account_name = "tfappprodio"
+container_name       = "terraform-state"
+key                  = "opex.io-web-profile-backend.external.tfstate"
```

Comment:
`io-web-profile-backend.opex.tfstate`?

Reply:
Looking at what `io-sign` does, I'd say `"opex.io-web-profile-backend.opex.external.tfstate"` to still include it under the `opex` umbrella. What do you think?

Reply:
with the one you suggested `opex` keyword is mentioned 2 times(as a prefix and before `.external`)

Reply:
Sorry - a typo. I meant `opex.io-web-profile-backend.external.tfstate`

Reply:
it's already as you mentioned